setup.cfg

```diff
@@ -235,6 +235,33 @@ console_scripts =
     dirac-wms-job-submit = DIRAC.Interfaces.scripts.dirac_wms_job_submit:main
     dirac-wms-jobs-select-output-search = DIRAC.Interfaces.scripts.dirac_wms_jobs_select_output_search:main
     dirac-wms-select-jobs = DIRAC.Interfaces.scripts.dirac_wms_select_jobs:main
+    dcd = DIRAC.Interfaces.scripts.dcd:main
+    dchgrp = DIRAC.Interfaces.scripts.dchgrp:main
+    dchmod = DIRAC.Interfaces.scripts.dchmod:main
+    dchown = DIRAC.Interfaces.scripts.dchown:main
+    dconfig = DIRAC.Interfaces.scripts.dconfig:main
+    dfind = DIRAC.Interfaces.scripts.dfind:main
+    dget = DIRAC.Interfaces.scripts.dget:main
+    dgetenv = DIRAC.Interfaces.scripts.dgetenv:main
+    dinit = DIRAC.Interfaces.scripts.dinit:main
+    dinput = DIRAC.Interfaces.scripts.dinput:main
+    dkill = DIRAC.Interfaces.scripts.dkill:main
+    dlogging = DIRAC.Interfaces.scripts.dlogging:main
+    dls = DIRAC.Interfaces.scripts.dls:main
+    dmeta = DIRAC.Interfaces.scripts.dmeta:main
+    dmkdir = DIRAC.Interfaces.scripts.dmkdir:main
+    doutput = DIRAC.Interfaces.scripts.doutput:main
+    dput = DIRAC.Interfaces.scripts.dput:main
+    dpwd = DIRAC.Interfaces.scripts.dpwd:main
+    drepl = DIRAC.Interfaces.scripts.drepl:main
+    dreplicas = DIRAC.Interfaces.scripts.dreplicas:main
+    drm = DIRAC.Interfaces.scripts.drm:main
+    drmdir = DIRAC.Interfaces.scripts.drmdir:main
+    dsetenv = DIRAC.Interfaces.scripts.dsetenv:main
+    dsize = DIRAC.Interfaces.scripts.dsize:main
+    dstat = DIRAC.Interfaces.scripts.dstat:main
+    dsub = DIRAC.Interfaces.scripts.dsub:main
+    dunsetenv = DIRAC.Interfaces.scripts.dunsetenv:main
```

Comment:
Do we need `dunsetenv` and `dsetenv`?

Reply:
Yes, it sets environment in the DIRAC session.

Reply:
I don't understand what that means? Sourcing the `diracosrc` sets variables and if people want more there is `export`. Why does DIRAC need a custom mechanism on top of that?

Reply:
Delete them already.